orchestra/analytics/latency.py

```diff
@@ -51,6 +53,8 @@ def work_time_df(projects, human_only=True, complete_tasks_only=True):
             human_only, complete_tasks_only)
         for project in projects))
     df = DataFrame(time_data)
+    df['start_datetime'] = df['start_datetime'].astype('datetime64[ns]')
```

Comment:
This comment still seems confusing to me—if pandas treats non-primitives as objects and we're storing the datetime objects in pandas originally, why the need to cast?